src/algebra/group/type_tags.lean

```diff
@@ -40,12 +46,24 @@ def to_mul : additive α ≃ α := of_mul.symm
 
 @[simp] lemma to_mul_symm_eq : (@to_mul α).symm = of_mul := rfl
 
+@[ext] protected lemma ext {x y : additive α} : x.to_mul = y.to_mul → x = y :=
+to_mul.apply_eq_iff_eq.mp
+
+protected lemma ext_iff {x y : additive α} : x = y ↔ x.to_mul = y.to_mul :=
+⟨congr_arg _, additive.ext⟩
+
```

Comment:
I think you can use `to_mul.injective.eq_iff` for this proof, but it's fine as is